vm-setup/roles/v1aX_integration_test/tasks/generate_templates.yml

```diff
@@ -32,3 +32,6 @@
       - cluster
       - controlplane
       - workers
+    environment:
+      CAPM3RELEASE: {{ CAPM3RELEASE }}
+      CAPIRELEASE: {{ CAPIRELEASE }}
```

Comment:
```suggestion
      CAPIRELEASE: "{{ CAPIRELEASE }}"
```

Reply:
I think running shell check locally would be better as there could be related issues.

Reply:
done